api/v1beta1/remotesecret_types.go

```diff
@@ -109,6 +109,12 @@ type RemoteSecret struct {
 
 	Spec   RemoteSecretSpec   `json:"spec,omitempty"`
 	Status RemoteSecretStatus `json:"status,omitempty"`
+	// Temporary optional field for data to upload during create/update RemoteSecret
+	// It is processed by Mutating Webhook and must not be persisted,
+	// to make sure (in a case if something happened with Webhook) it is constrained
+	// by MaxItems=0, made as array of objects since it looks like it does not work for map
+	//+kubebuilder:validation:MaxProperties=0
```

Comment:
Is this comment still accurate?

Reply:
yes

Reply:
It's kind of a trick, this data should not pass through to the kube API/controller, is should be read and cleaned up in webhook.